tests/core/test_actions.py

```diff
@@ -344,6 +367,164 @@ async def test_remote_action_with_template_param(
     ]
 
 
+@pytest.mark.parametrize(
+    ("event", "expected_event"),
+    (
+        ({"event": "entities", "timestamp": 1621590172.3872123}, EntitiesAdded),
```

Comment:
How about changing this test to use event objects and then call `as_dict()` on them instead of putting the output payload directly? This would make the test more robust in case somebody decides to change how an event is serialized.

e.g. something along these lines:

```python
events = [ActionExecuted(action_name="Hi Anca", timestamp=1234)]
events = [e.as_dict() for e in events]
validate(events)
```